### ISSUE TITLE
gcroots for booted system and running system are not created

### ISSUE THREAD

**Issue by sky1e** [nixos-bsd/nixbsd#18]
When you run nix-collect-garbage the running system commits suicide and deletes itself:
![image](https://github.com/nixos-bsd/nixbsd/assets/5225017/d361e0c0-d6a2-4140-83b2-c101fef0993b)
`/nix/store/` is now empty.

**Comment by sky1e** [comment]
This can be fixed manually with
```bash
ln -s /run/booted-system booted-system
ln -s /run/current-system current-system
```

**Comment by sky1e** [comment]
Fixed by f5f577781cfb0a9f7cdf6785e33b754f3efe84de
